frontend/src/exploration/questions/Questions.tsx

```diff
@@ -1,20 +1,122 @@
-import React from 'react';
-import { Box, makeStyles } from '@material-ui/core';
-import { red } from '@material-ui/core/colors';
+import React, { useState } from 'react';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  FormGroup,
+  makeStyles,
+  Theme,
+  Typography,
+} from '@material-ui/core';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import { createStyles } from '@material-ui/core/styles';
+import ExplorationStore from '../../stores/exploration/ExplorationStore';
+import useService from '../../dependency-injection/useService';
+import explorationQuestions from '../../stores/exploration/ExplorationQuestions';
+import { ExplorationAnswer } from '../../stores/exploration';
 
-const useStyle = makeStyles(() =>
+const useStyle = makeStyles((theme: Theme) =>
   createStyles({
-    container: {
-      backgroundColor: red[500], // TODO change/remove
-      height: '75vh', // TODO change/remove
+    heading: {
+      fontSize: theme.typography.pxToRem(15),
+      fontWeight: theme.typography.fontWeightRegular,
+    },
+    formControl: {
+      margin: theme.spacing(3),
     },
   })
 );
 
+// Interface to keep track of checked answers in component state.
+interface ExplorationSelection {
+  answer: ExplorationAnswer;
+  checked: boolean;
+}
+
+// Map answers to string made of questionIndex and answerIndex, in order to add and
+// remove them from the explorationStore in handleChange.
+const selections: Record<string, ExplorationSelection> = {};
+explorationQuestions.forEach((q, qIndex) => {
+  q.answers.forEach((a, aIndex) => {
+    selections[`explorationAnswer-${qIndex}-${aIndex}`] = {
+      answer: a,
+      checked: false,
+    };
+  });
+});
+
```

Comment:
Tracking of checked answers can be done by checking if they are in the store state.
```suggestion
```
See next suggestions